Routing API Updates of Sept 30th (#387)
MicrosoftDocs/bingmaps-docs: BingMaps/includes/bing-maps-local-insights-api-retirement.md

> [!NOTE]
> **Bing Maps Local Insights API retirement**
>

> > The Bing Maps Local Insights API is deprecated and was retired on 9/30/2024. To avoid service disruptions, all implementations using Bing Maps Local Insights API will need to be updated to use [Azure Maps](https://azure.microsoft.com/products/azure-maps) by 9/30/2024.

Reviewer: I'd like this better if we had a specific Azure Maps API to point to here... Here is a suggestion for alternative wording:

```suggestion
> The Bing Maps Local Insights API was retired on 9/30/2024 and is no longer supported. All implementations using Bing Maps Local Insights API can be updated to use [Azure Maps](https://azure.microsoft.com/products/azure-maps).
```

Reviewer 2: I'm not sure there is an Azure Maps replacement for this API actually.
Also, it is officially retired - per the e-mail thread - "we can remove it from the documentation" 
It may be simpler to do that?

Reviewer: Yes, my suggestion would be to remove it from the docs since it has been retired (since 9/30/2024). 

references to it will need to be removed as well, for example:

**\BingMaps\getting-started\bing-maps-dev-center-help\understanding-bing-maps-transactions.md**

Line  90: |RESTLocalInsights| REST Services | Yes<sup>3</sup> | Any time a [Local Insights](../../rest-services/routes/local-insights.md) URL request is made, one (1) transaction is counted.|

**\BingMaps\rest-services\routes\index.md**

Line 35: |[Local Insights](local-insights.md)| Returns local business entities centered at a location within a maximum driving time or distance traveled.|  

**\bingmaps-docs\BingMaps\rest-services\routes\local-insights-data.md**

Remove file and reference to it from the TOC (\bingmaps-docs\BingMaps\rest-services\toc.yml)

**\bingmaps-docs\BingMaps\rest-services\routes\local-insights.md**

Remove file and reference to it from the TOC (\BingMaps\rest-services\toc.yml) and references to it from other markdown files:

- \BingMaps\getting-started\bing-maps-dev-center-help\understanding-bing-maps-transactions.md (Line 90)
- \BingMaps\rest-services\routes\index.md (Line 35)
- \BingMaps\rest-services\routes\local-insights-data.md (Line 22)
